### PR TITLE
Add Jenkinsfile

### PR DESCRIPTION
**Pull request by quantranhong1999**

### CODE REVIEW

**Review by chibenwa — COMMENTED** [review]
It would be nice to have instructions to run this JenkinsFile locally IMO

**Review by Arsnael — APPROVED** [review]

**Review by Arsnael — APPROVED** [review]

**Review by chibenwa — APPROVED** [review]

File: dockerfiles/jenkins-runner/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+# Install git, docker, sdkman
+RUN apt-get update && \
+  apt-get install -y wget git unzip zip apt-transport-https ca-certificates curl gnupg2 software-properties-common && \
+  curl -fsSL https://download.docker.com/linux/$(. /etc/os-release; echo "$ID")/gpg > /tmp/dkey; apt-key add /tmp/dkey && \
+  add-apt-repository \
+    "deb [arch=amd64] https://download.docker.com/linux/$(. /etc/os-release; echo "$ID") \
+    $(lsb_release -cs) \
+    stable" && \
+  apt-get update && \
+  apt-get -y install docker-ce &&  \
```
Comment (vttranlina) [inline dockerfiles/jenkins-runner/Dockerfile:12]: Q: 
Why do we need docker (docker in docker) here?
I think we need only the sbt?

Comment (Arsnael) [reply dockerfiles/jenkins-runner/Dockerfile:12]: the gatling integration tests need to have a tmail memory docker container running to be able to execute :)

Comment (vttranlina) [reply dockerfiles/jenkins-runner/Dockerfile:12]: understood, thanks

**Review by vttranlina — APPROVED** [review]

**Comment by Arsnael** [comment]
The Jenkinsfile technically is good so I merge it.

However https://github.com/linagora/james-gatling/pull/129 got merged before it got a chance to run on the CI and I detected a problem with it.

Something is wrong with the private key for JMAP. I could detect it on the slave by checking the logs of the failed start of the tmail container.

**Pull request merged**